Classes/Fusion/SelectOptionCollectionImplementation.php

```diff
@@ -39,9 +44,9 @@ public function evaluate()
     /**
      * @return array|\Traversable
      */
-    private function getCollection()
+    private function getItems()
     {
-        return $this->fusionValue('collection');
+        return $this->fusionValue('items');
```

Comment:
Just wondering: Why did you rename that? Doesn't that lead to a needless breaking change?

Reply:
This could just be a search and replace issue while fixing the deprecations in the Neos.Fusion:Loop integrations - switchting from collection to items 🙈 Will change it back!

Reply:
So, my intention was to have the form builder using the same nomenclature as neos fusion prototypes and change the deprecated usage of `collection` to `items`. I could revert it. What do you think? Would this be okay to document in the change log as a necessary migration / code adjustment when upgrading?

Reply:
OK, thanks for the explanation. As long as it does not require a db migration I'd personally be fine with both